client/state/current-user/constants.js

```diff
@@ -0,0 +1,3 @@
+export default {
+	DWPO: 'calypso_domains_with_plans_only'
```

Comment:
`DWPO` seems too cryptic, IMHO. From what I've seen, even some people in SDIV don't know what it means, not to mention a new Calypso developer :/ Maybe you could just write it as `DOMAINS_WITH_PLANS_ONLY` (to signify it's a const and avoid naming conflicts with `domainsWithPlansOnly`)?